hive/Dockerfile

```
@@ -22,13 +22,9 @@ RUN microdnf update && \
     microdnf install tar gzip zip which shadow-utils && \
```

`shadow-utils` is not used anymore and can be removed.

```suggestion
    microdnf install tar gzip zip which && \
```